packages/webcert/src/components/commonLayout/CommonLayout.tsx

```diff
@@ -21,16 +22,17 @@ interface Props {
   header?: React.ReactNode
   subHeader?: React.ReactNode
   noPadding?: boolean
+  hasSidePanel?: boolean
 }
 
-const CommonLayout: React.FC<Props> = ({ header, subHeader, children, noPadding = false }) => {
+const CommonLayout: React.FC<Props> = ({ header, subHeader, children, hasSidePanel = false, noPadding = false }) => {
   return (
     <Root>
       <CustomTooltip placement="top" />
       {header && <NoFlexGrow>{header}</NoFlexGrow>}
       {subHeader && <NoFlexGrow>{subHeader}</NoFlexGrow>}
-      <Content>
-        <div className={`ic-container ${noPadding === false && 'iu-pt-500'}`}>{children}</div>
+      <Content className={classnames({ 'ic-container': hasSidePanel })}>
+        {!hasSidePanel ? <div className={`ic-container ${noPadding === false && 'iu-pt-500'}`}>{children}</div> : children}
```

Comment:
Maybe turn the statement around so it says hasSidePanel instead of !hasSidePanel to avoid confusion?